source/cmake_modules/ThirdParty.cmake

```diff
@@ -22,8 +22,8 @@ set(BUILD_EXAMPLES OFF CACHE BOOL "don't build example of librabbimq-c")
 set(BUILD_TOOLS OFF CACHE BOOL "don't build tool of librabbimq-c")
 set(BUILD_TESTS OFF CACHE BOOL "don't build tests of librabbimq-c")
 set(Rabbitmqc_INCLUDE_DIRS "${CMAKE_SOURCE_DIR}/third_party/librabbitmq-c/librabbitmq/")
-add_subdirectory(third_party/librabbitmq-c)
-add_subdirectory(third_party/SimpleAmqpClient)
+add_subdirectory(third_party/librabbitmq-c EXCLUDE_FROM_ALL)
```

Comment:
So for those we only use `.h` if I understand correctly?

Reply:
`EXCLUDE_FROM_ALL` is used to inform the makefile to Ignore the install command into the subdirectory. 
By default, this is allowed. So, when you create the .Deb package and you install it, `install` command into third party are executed. 
But we only use `third party` for static compilation. We don't want to deploy the lib (like header file) on target machine.
It is just more clean, we only install what we need.

Reply:
From the doc, I understand that it prevents build also (not ony install).
That's fine for install and I guess we will know soon enough if we miss some compiling :)